### DISCUSSION TITLE
Add Area information

### DISCUSSION THREAD

**Discussion by A35wc** [code4recovery/12-step-meeting-list#1109] [Q&A] [answered]
Is there a way to add an Area field?  We share our website between the 2 areas in MN, so searching and filtering meetings on District returns meetings for the district in both areas.

**Comment by joshreisner** [comment, marked as answer]
Districts can be hierarchical. You can go to the Meetings > Districts menu and make it 

* Area 35
  * District 01
  * District 02
* Area 36
  * District 01
  * District 02

Does that help?

**Reply by A35wc** [reply]
Thanks Josh, I think it does.  Once I create the new parent can I create new districts under the parents to move the meeting to the appropriate districts - will the hierarchical structure create new values?

**Reply by joshreisner** [reply]
the flow i would follow is i would create the Areas as districts, and then edit the individual Districts to specify the correct area parent. by editing, you should preserve the existing relationships with groups

**Reply by A35wc** [reply]
Got it.  That will at least get me half way there - right now the districts are comingled across areas.
